doc/release-notes/0.51/0.51.md

```diff
@@ -137,6 +137,27 @@ on x64 platforms, but builds with OpenJDK 17 and later also require more stack s
 <td valign="top">Avoid using the <tt>-XX:+ShowHiddenFrames</tt> option with OpenJDK 18 and later.</td>
 </tr>
 
+<tr>
+<td valign="top">N/A </td>
+<td valign="top">JVM fails to load AOT compiled code from portable shared class cache created on IBM z17.</td>
+<td valign="top">Linux on IBM Z®</td>
+<td valign="top">When docker container image with IBM Semeru Runtime is created
+on system running on IBM z17 hardware and deployed on older IBM Z® hardware,
+JVM will fail to validate AOT compiled code which would cause JVM to start JIT
+compiling methods. This issue can impact increased CPU usage driven by JIT
+compilation activities and/or slow application star-up. Outside of docker
+container if portable shared class cache is enabled explicitly by using the
+<tt>-XX:+PortableSharedCache</tt> option, similar issue can be seen when such
+shared class cache is populated with portable AOT code on IBM z17 hardware and
+used on older IBM Z® hardware.</td>
+<td valign="top">If you are deploying application containers using IBM Semeru
```

Comment:
There is another IBM Semeru Runtime here.

Reply:
Replaced with Eclipse OpenJ9 in https://github.com/eclipse-openj9/openj9/compare/5039b9d7e9b766d54e0d9727becdb54d9a966720..cff69dc5e1134d1b1051fdc25bf69de7ec7a88e5